src/server/render.tsx

```diff
@@ -310,6 +310,7 @@ export async function suspenseRender(
     function render() {
       try {
         body = renderToString(vnode);
+        return body;
```

Comment:
In the "catch a promise path" this should return an empty string.

Reply:
should i change the typings to RendererModule.render to return only string and not null then?

Reply:
Yes